src/z3c/saconfig/README.rst

```diff
@@ -333,7 +334,7 @@ Configuration using ZCML
 A configuration directive is provided to register a database engine
 factory using ZCML.
 
-  >>> from cStringIO import StringIO
+  >>> from six import StringIO
```

Comment:
Probably you should use from io import BytesIO and be sure that you are passing bytes to it.
See this patch as an example https://github.com/plone/plone.supermodel/commit/1d02b25d02b825cd64fa95760b0abee1b5055e7e#diff-bc1d78c2a92695a5b3d8a63f5ea6ab0fR16